### PR TITLE
use of label in DID list

### PR DESCRIPTION
**Pull request by manonthemat**
Signed-off-by: Matthias Sieber <matthias.sieber@loxeinc.com>

### CODE REVIEW

**Review by cjhowland — APPROVED** [review]
Looks good to me!

**Comment by manonthemat** [comment]
@cjhowland, thank you for the review.
I'm requesting a re-review as I added the label of a DID in the dropdown and active DID state after attending today's workshop - which by the way, was phenomenal.

**Comment by manonthemat** [comment]
@frostyfrog when I requested another review after pushing to the already approved PR, you got removed from the reviewers. :thinking:

**Comment by manonthemat** [comment]
This is what it looks like.

DID list with labels

File: src/renderer/components/AgentBase.vue [hunk 1]
```diff
@@ -1,35 +1,21 @@
...
-          <el-option
-            v-for="did in dids"
-            :key="did.did"
-            :label="did.did"
-            :value="did">
+      <a class="navbar-brand" href="#">{{ connection.label }}</a>
+      <el-form :disabled="dids.length === 0">
+        <el-select v-model="active_did" no-data-text="No DIDs found" filterable placeholder="Activate DID">
+          <el-option v-for="did in dids" :key="did.did"
+            :label="did.metadata?.label ? `${did.metadata.label} (${did.did})` : did.did" :value="did">
```
Comment (cjhowland) [inline src/renderer/components/AgentBase.vue:18]:
```suggestion
            :label="'metadata' in did && 'label' in did.metadata ? `${did.metadata.label} (${did.did})` : did.did" :value="did">
```

**Review by cjhowland — CHANGES_REQUESTED** [review]
Hi Matthias! Thanks for attending the workshop and making this useful addition.

It looks as though the syntax `did.metadata?.label` is not supported and the toolbox cannot compile for me locally. Borrowing the syntax from the conditional statement in the `get_name` method in `DidList.vue` seems to fix it.

**Review by cjhowland — APPROVED** [review]

**Pull request merged**